roles/openshift_metering/README.md

```diff
@@ -0,0 +1,49 @@
+# OpenShift Metering
+
+This role installs the OpenShift [Metering](https://github.com/operator-framework/operator-metering), which supports metering operators and applications in Kubernetes and producing reports on this metered information.
+
+### **NOTE: This component is unsupported in OCP at this time.**
```

Comment:
Should ansible throw a warning when `deployment_type=='openshift-enterprise'`?

Reply:
I don't really know.